src/lib/viewers/controls/rotate/RotateControl.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import IconRotate24 from '../icons/IconRotate24';
+import './RotateControl.scss';
+
+export type Props = {
+    onRotateLeft: () => void;
+};
+
+export default function RotateControl({ onRotateLeft }: Props): JSX.Element {
+    return (
+        <div className="bp-RotateControl">
+            <button
+                className="bp-RotateControl-button"
+                data-testid="bp-RotateControl-out"
```

Comment:
What is `-out` here?

Reply:
This was copied from ZoomControls, will remove!